website/js/content.js

```diff
@@ -272,21 +294,56 @@ const helpContentResearch = [
                 ]
             },
             {
-                name: "In Silico Prior Probabilities of Pathogenicity",
+                name: "What are In Silico Prior Probabilities of Pathogenicity?",
                 contents: require("../content/help/research/insilico/insilico-pred.md")
             },
             {
-                name: "CRAVAT/MuPIT Interactive Protein Structure Viewer",
+                name: "How do I interpret the CRAVAT/MuPIT Interactive Protein Structure Viewer?",
                 id: "cravat-mupit-3d-protein-view",
                 contents: require("../content/help/research/cravat-mupit.md")
             },
             {
-                name: "Literature Search Results",
+                name: "How does the BRCA Exchange provide Literature Search Results for variants?",
                 contents: require("../content/help/research/literature-search.md"),
                 isBeta: true
             },
+            {
+                name: "What genome build is this site using?",
+                id: "cravat-mupit-3d-protein-view",
+                contents: require("../content/help/research/what-genome-build-is-this-site-using.md")
+            },
+            {
+                name: "How is the data on this site updated?",
+                contents: require("../content/help/research/how-is-the-data-on-this-site-updated.md"),
+                isBeta: true
```

Comment:
if you remove this line `isBeta: true` it will remove the beta tag from this question